docker-compose.yaml

```diff
@@ -49,7 +53,8 @@ services:
         labels:
             org.label-schema.name: ML Serve
             org.label-schema.description: Alphan Atinok's Nuclei Detector (powered by Ray Serve)
-        
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
```

Comment:
Oh wow never seen this before. `host-gateway` must be a special token that lets `host.docker.internal` adjust its `/etc/hosts` to reach the host instead of the container?

Reply:
@nutjob4life yup, this is so docker could reach the Solr service running on the host! Don't know if this is a good  practice though.